tools/run_sanitizer.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+SANITIZER=$1
+
+if [ "$#" -eq 2 ]; then
+  SOURCE=$2
+else
+  SOURCE=$(cd $(dirname $BASH_SOURCE); pwd)/..
+fi
+
+WORKING_DIR=$PWD
+BUILD=$WORKING_DIR/build-sanitizer
+INSTALL=$WORKING_DIR/install-sanitizer
+
+cd $WORKING_DIR
+mkdir $BUILD
+mkdir $INSTALL
+cd $BUILD
+# We need to disable ctest integration of gtest-based tests since gtest_discover_tests
+# fails due to missing (asan) preload.
+cmake -DWITH_CTEST=Off -DSANITIZE_$SANITIZER=On -DCMAKE_INSTALL_PREFIX=$INSTALL $SOURCE
+make -j12 install
```

Comment:
```suggestion
make -j$(nproc) install
```

Not ideal as this is Linux only. An alternative would be to use Ninja which by default uses all available cores.

Reply:
Why not `make -j`?

Reply:
Didn't know `-j` could be used like that.

Reply:
```suggestion
make -j install
```